tensorflow/python/ops/image_ops.py

```diff
@@ -34,6 +34,7 @@
 @@resize_bilinear
 @@resize_nearest_neighbor
 @@resize_image_with_crop_or_pad
+@@resize_image_aspect_with_pad
```

Comment:
Why is "aspect" in the name? I'd just call this `resize_image_with_pad`

Reply:
I think the name should highlight the difference from the other `resize_*` methods, all of which distort the image if the new dimensions don't match the aspect ratio.

Reply:
`resize_image_with_crop_or_pad` doesn't distort either. I think calling it `resize_image_with_pad` would be better.